### ISSUE TITLE
[UI] inputType for "arrays"?

### ISSUE THREAD

**Issue by Remi-Gau** [ReproNim/reproschema#404]
@sanuann @satra 

This might be cryptic way to describe this but for MRI and PET we are getting a lot items that require answer in terms of 

`X= ; Y= ; Z= ` 

Very often for voxel dimension, image resolution...

It does not seem like a good idea to have 3 separate "float" items for this.

Did I miss it or is this something covered by the schema at the moment on the UI part in terms of possible inputType?

**Comment by satra** [comment]
we do something like this for language spoken question.

however, structured responses (i.e. like python dict) are definitely not supported at this point. would be interesting to consider or in terms of grouping fields and exposing them in one go.

**Comment by sanuann** [comment]
for languages spoken, we have a multiselect dropdown. 
There is one widget called `MultiTextInput` which is used for say given, middle and last name. Perhaps can do a similar one for the above use case.. `MultiFloatInput`?

**Comment by Remi-Gau** [comment]
yup the `MultiFloatInput` idea sounds very similar to what I had in mind. This would be perfect! :+1:

**Comment by sanuann** [comment]
can you please add a TODO in the issue list for this?
